aiopyql/table.py

```diff
@@ -561,6 +616,7 @@ async def update(self, where: dict, **kw):
             return result
         except Exception as e:
             self.log.exception(f"Exception updating row for {self.name}")
+            print("\nQuery produced: ", query)
```

Comment:
@AdhTri-001 
if printing is required, or deemed helpful, then it is better to use self.log.debug() or self.log.warning()

Reply:
Ok